src/index.js

```diff
@@ -88,7 +88,13 @@ class ZopfliPlugin {
           cb();
         });
       }, callback);
-    });
+    };
+
+    if (compiler.hooks) {
+      compiler.hooks.emit.tapAsync(this.constructor.name, processAssets);
```

Comment:
Oh, sorry, please use `ZopfliPlugin` string instead `this.constructor.name`

Reply:
Fixed